share/dkg/pedersen/dkg.go

```diff
@@ -10,11 +10,14 @@
 package dkg
 
 import (
+	"crypto/cipher"
+	"crypto/rand"
 	"errors"
 	"fmt"
+	"io"
 
-	"go.dedis.ch/kyber/v3"
-
+	"github.com/PizzaWhisperer/kyber/util/random"
```

Comment:
That's the thing with Go, it's not easy to fork ^^ Normally with the Go module I think you should be able to use the kyber path...
If it's not working, then try to do 2 PR: first for that random package and then for DKG